docs/topics/auth/default.txt

```diff
@@ -1280,9 +1280,19 @@ implementation details see :ref:`using-the-views`.
     Allows a user to reset their password by generating a one-time use link
     that can be used to reset the password, and sending that link to the
     user's registered email address.
-
-    If the email address provided does not exist in the system, this view
-    won't send an email, but the user won't receive any error message either.
+    
+    This view will send an email only if the following conditions are met:
+    
+    #. The email address provided exists in the system.
+    #. The requesting user is active (``User.is_active`` is ``True``).
+    #. The requesting user has a usable password.
```

Comment:
It's not the _requesting user_ per se, but the _User account_ found/being looked up (the _requested user_ I suppose).

ie: I can ask for a password reset email for anyone, as the requesting user (an `AnonymousUser`, which doesn't _have_ `is_active`), as long as I know their email address.

Reply:
Good point, will update.

Reply:
Changed in 185884b997.